HW_35/script.js

```diff
@@ -0,0 +1,74 @@
+var users = [
+    {
+        name: "Harry Felton",
+        phone: "(09) 897 33 33",
+        email: "felton@gmail.com",
+        animals: ["cat"],
+        cars: ["bmw"],
+        hasChildren: false,
+        hasEducation: true,
+    },
+    {
+        name: "May Sender",
+        phone: "(09) 117 33 33",
+        email: "sender22@gmail.com",
+        hasChildren: true,
+        hasEducation: true,
+    },
+    {
+        name: "Henry Ford",
+        phone: "(09) 999 93 23",
+        email: "ford0@gmail.com",
+        cars: ["bmw", "audi"],
+        hasChildren: true,
+        hasEducation: false,
+    },
+];
+// 1. Создать строку из имен пользователей через запятую
+function getStringNames(array) {
+    var stringNames = [];
+    array.forEach(function (item) {
+        stringNames.push(item.name);
+    });
+    console.log(stringNames);
+    return stringNames;
+}
+getStringNames(users);
+// 2. Посчитать общее количество машин у пользователей
+function countNumOfCars(array) {
+    var numOfCars = 0;
+    array.forEach(function (item) {
+        if (item.cars !== undefined) {
+            numOfCars += item.cars.length;
+        }
+    });
+    console.log(numOfCars);
+}
+countNumOfCars(users);
+// 3. Создать функцию, которая бы принимала массив пользователей и отфильтровывала пользователей на наличие образования
+function filterArrayEducation(array) {
+    var arrFilterEducation = array.filter(function (item) {
+        return item.hasEducation === true;
```

Comment:
Cleaner way to do the same is:

`array.filter(({hasEducation}) => hasEducation)`